Modules/Scripted/Home/Home.py

```diff
@@ -201,6 +201,9 @@ def fromDict(cls, data):
       logging.warning("Annotation data is missing the 'name' key. Defaulting to '%s'." % annotationName)
     annotation.markup.SetName(annotationName)
 
+    # annotation is initialized at this point, so trigger an update
+    annotation.update()
```

Comment:
Instead of triggering the update from this method, I suggest to consider the following changes:

* trigger update from the `add` method if `setCurrent` is `True`:

https://github.com/BICCN/cell-locator/blob/e70b3101559ea143d5b8fd1bcbe0de9132f612b9/Modules/Scripted/Home/Home.py#L326-L332


* and reset current when the complete set of annotations is updated:

https://github.com/BICCN/cell-locator/blob/e70b3101559ea143d5b8fd1bcbe0de9132f612b9/Modules/Scripted/Home/Home.py#L627-L634

Reply:
It's important to make sure that _all_ annotations are updated, not just the current one; so I instead update each annotation from `setAnnotations`. 

This is done in d07c652.